package.json

```diff
@@ -37,8 +37,15 @@
   "scripts": {
     "build": "turbo run build",
     "clean": "git clean -fdx -e '.env*'",
-    "db:migrate": "bun run -F @hebo/database db migrate dev",
-    "db:reset": "bun run -F @hebo/database db migrate reset --force",
+    "db:api:generate": "bun run -F @hebo/api db generate",
```

Comment:
Intermediary db:api*/db:auth* commands seem unnecessary and don't align with existing conventions. Just inline into the "master" command.